### ISSUE TITLE
does not accept `without-match`

### ISSUE THREAD

**Issue by NightMachinary** [Genivia/ugrep#29]
```
❯ ugrep --binary-files=without-match -Q                                                                                      
ugrep: invalid argument --binary-files=TYPE, valid arguments are 'binary', 'without-match', 'text', 'hex', and 'with-hex'
Usage: ugrep [OPTIONS] [PATTERN] [-f FILE] [-e PATTERN] [FILE ...]
Try 'ugrep --help' for more information.

❯ ugrep --binary-files='without-match' -Q                                                                                    
ugrep: invalid argument --binary-files=TYPE, valid arguments are 'binary', 'without-match', 'text', 'hex', and 'with-hex'
Usage: ugrep [OPTIONS] [PATTERN] [-f FILE] [-e PATTERN] [FILE ...]
Try 'ugrep --help' for more information.

❯ ugrep --version                                                                                                            
ugrep 2.1.1 x86_64-apple-darwin19.0.0 +avx +pcre2_jit +zlib +bzip2 +lzma
License BSD-3-Clause: <https://opensource.org/licenses/BSD-3-Clause>
Written by Robert van Engelen and others: <https://github.com/Genivia/ugrep>
```

**Comment by genivia-inc** [comment]
Thanks for the feedback! There is a typo in `ugrep.cpp`: "without-matches" instead of "without-match". Option `-I` works of course, where `--binary-files=without-match` should be the same. Thanks for spotting that.

**Comment by genivia-inc** [comment]
Fixed in v2.1.2.
